packages/browser-destinations/src/destinations/adobe-target/upsertProfile/index.ts

```diff
@@ -10,11 +10,11 @@ const action: BrowserActionDefinition<Settings, Adobe, Payload> = {
   platform: 'web',
   defaultSubscription: 'type = "identify"',
   fields: {
-    userId: {
+    mbox3rdpartyid: {
```

Comment:
```suggestion
    mbox_third_party_id: {
```
or
```suggestion
    user_id: {
```

As long as the display label is Mbox 3rd Party ID, the key can be either of the above! We use `user_id` in Adobe Target Cloud cc @rvadera12

Reply:
I used user_id instead of userId since I noticed most of the action destination fields use snake case.